src/components/FilterList.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const DefaultItem = ({ serviceName, departmentName }) => (
+  <div>
+    <h3>{serviceName}</h3>
+    <p>{departmentName}</p>
+  </div>
+);
+
+const FilterList = ({
+  items = [],
+  renderItem = props => <DefaultItem {...props} />
+}) => console.log(items) || <ul>{items.map(renderItem)}</ul>;
```

Comment:
we will want to remove this console.log. We also don't want to use a ul here, since the default item is not an `li`. 

There is a pattern for that, see here for the [implementation](https://codesandbox.io/s/react-holiday-day-15-st2l3).

With a video explanation https://twitter.com/chantastic/status/1205904268881846272?s=20

Reply:
@martypowell I think I did this correctly. Im not sure I totally followed what he was saying but take a look and see if this is more in line with what you wanted.